pkg/git/git.go

```diff
@@ -0,0 +1,61 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+package git
```

Comment:
Empty line between comment and `package`, otherwise it would be considered as the package comment for `godoc` I think.
```suggestion
// SPDX-License-Identifier: Apache-2.0

package git
```

Reply:
Done, thanks!